README.md

```diff
@@ -17,55 +17,114 @@ For further details about the method, please refer to the [PDF documentation](ht
 
 ```{sh}
--DCMAKE_CXX_COMPILER=/Users/username/.local/homebrew/opt/llvm/bin/clang++
--DOMP_ROOT=/Users/username/.local/homebrew/opt/llvm/
--DCXX_ROOT=/Users/username/.local/homebrew/opt/llvm
--DHDF5_ROOT=/Users/username/.local/homebrew/opt/hdf5
+$ brew install cmake fftw llvm
 ```
 
-On an ARM Mac, you will need to install the x86 version of Homebrew.
-To do so, use the following commands:
+On an ARM Mac, you will need to install the x86 version of Homebrew:
 
 ```{sh}
 $ arch -x86_64 zsh
 $ arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-$ arch -x86_64 /usr/local/bin/brew install llvm hdf5
+$ arch -x86_64 /usr/local/bin/brew install cmake fftw llvm
+```
+
+</details>
+
+<details>
+<summary><img src="https://github.com/EgoistDeveloper/operating-system-logos/blob/master/src/24x24/WIN.png"/> Windows prerequisite setup</summary>
+
+TDMS was developed on, and has been extensively tested on linux.
+Support for Windows is quite new and experimental (please [report](https://github.com/UCL/TDMS/issues/new/choose) any issues you encounter!).
+
+It might be more straightforward to use the [Windows subsystem for Linux (WSL2)](https://learn.microsoft.com/en-gb/windows/wsl/install), or set up an linux virtual machine.
+
+However, TDMS _can_ be compiled natively on Windows.
+This has been tested Windows 10 with PowerShell.
+
+Assuming you don't already have them, you'll need to download and install:
+
+* [MATLAB](https://www.mathworks.com/products/matlab.html),
+* [Visual Studio](https://visualstudio.microsoft.com/vs/community/) and be sure to select the C++ kit,
+* [CMake](https://cmake.org/download/),
+* and [FFTW](https://www.fftw.org/install/windows.html).
+
+Potentially the simplest way to get FFTW is via [conda](https://anaconda.org/conda-forge/fftw):
+
+```{pwsh}
+PS> conda install -c conda-forge fftw --yes
+```
+
+You'll need to ensure the paths to FFTW and MATLAB (the locations of `fftw3.dll` and `libmex.dll` respectively) are in the `env:Path`.
+
+These can be found, e.g. by
+```{pwsh}
+PS> conda list fftw # assuming you installed via conda
+PS> which.exe MATLAB
+```
+Which should return something like `C:\Program Files (x86)\MATLAB\R20XXx\bin\matlab` and maybe `C:\ProgramData\envs\base\bin`.
+If you downloaded FFTW and created `fftw3.dll` with `lib.exe`, you just need to know where you saved it.
+
+You can append the paths:
+
+```{pwsh}
+PS> $env:Path += ";C:\Program Files (x86)\MATLAB\R20XXx\bin\;C:\ < wherever fftw3.dll is >"
 ```
+
+Which will help Windows locate `.dll` files later.
+For all following instructions, you'll have to substitute our mentions of `tdms` with `tdms.exe` and `$` is used to denote a command prompt which, in PowerShell would look like `PS>`
+
 </details>
 
+------
+
+You'll need to download and install [MATLAB](https://www.mathworks.com/products/matlab.html), and take note where the headers are installed.
```

Comment:
If we're saying this, we might as well move it into the install list above (like when we say you need VSCode _and_ then do this other thing while you install it).

```md
Assuming you don't already have them, you'll need to download and install:

* [MATLAB](https://www.mathworks.com/products/matlab.html), and **take note where the headers are installed**
```

Reply:
Ah but this one is common to all three `<details>` sections. As in: do all of the things, and then get MATLAB.

Reply:
Perfectly happy to put it up into the sections though. We also say something similar _before_ all of the `<details>` blocks.